src/stress_analyzer.py

```diff
@@ -0,0 +1,157 @@
+import argparse
+from remote_execution import *
+
+### Functions below are for getting initial container dictionaries
+
+# Function that calls on the correct get_container function based on TYPE
+def get_container_ids(vm_ips, services, stress_type):
```

Comment:
Let us call this stress_policy instead of stress_type. Confusing since we actually do have different types of stresses.